src/ui/Header/Header.jsx

```diff
@@ -154,9 +154,11 @@ const Main = ({
     setSearchIsActive(false);
     setBurger('');
     // remove active menu when we have no pathname which means we hit logo to go home
-    if (!pathname) {
+    //remove the lang route in order to check if empty
+    //setActiveItem as pathname when pathname changed
+    if (!pathname.split('/').slice(2).join('/')) {
```

Comment:
@dobri1408
1. I think this will crash if the pathname is null or undefined
2. Not all websites will have a language root `/en/`